assets/js/components/Beneficiaries.vue

```diff
@@ -271,17 +328,37 @@ export default Vue.extend({
       /*
        *
        */
-      bentered.append("title").text(
+      bentered.attr("title",
```

Comment:
Since we now have the tooltip there is no need for the title tag and the extra info about an element should be in the title attribute. I have also removed the tag in the title in the latest commit and everything looks and works good